models/ht_neuron.h

```diff
@@ -59,48 +59,72 @@
    - No hard reset, but repolarizing potassium current.
    - AMPA, NMDA, GABA_A, and GABA_B conductance-based synapses with
      beta-function (difference of two exponentials) time course.
+   - Voltage-dependent NMDA with instantaneous blocking and two-stage
+     unblocking as described in [1].
    - Intrinsic currents I_h (pacemaker), I_T (low-threshold calcium),
      I_Na(p) (persistent sodium), and I_KNa (depolarization-activated
      potassium).
+   - Several apparent typographical errors in the descriptions of
+     the intrinsic currents were fixed, hopefully in a meaningful
+     way.
 
-   In comparison to the model described in the paper, the following
-   modifications were mare:
+   NMDA conductances is modeled as follows
 
-   - NMDA conductance is given by g(t) = g_peak * m(V), where
+     g_NMDA(t, V) = m(V, t) g(t)
 
-       m(V) = 1 / ( 1 + exp( - ( V - NMDA_Vact ) / NMDA_Sact ) )
+   where
 
-     This is an approximation to the NMDA model used in [2].
+     g(t) = g_peak ( e^(-t/tau_1) - e^(-t/tau_2) )
+               / ( e^(-t_peak/tau_1) - e^(-t_peak/tau_2) )
 
-   - Several apparent typographical errors in the descriptions of
-     the intrinsic currents were fixed, hopefully in a meaningful
-     way.
+     t_peak = tau_2 tau_1 / ( tau_2 - tau_1 ) ln( tau_2 / tau_1 )
+
+     m(V, t) = a(V) m_fast*(V, t) + ( 1 - a(V) ) m_slow*(V, t)
+     a(V)    = 0.51 - 0.0028 V
```

Comment:
There is an apparent problem with the homogeneity, here... I suppose 0.0028 is in mV^{-1}, but could you specify it explicitely? (maybe introduce a named constant).
Shouldn't the user want to be able to set these two values? If not (which would be surprising to me), would you mind explaining their origin?